### ISSUE TITLE
Svelte kit

### ISSUE THREAD

**Issue by kamalkech** [bestguy/sveltestrap#245]
Sveltestrap is compatible with Sveltekit ??

i test with two case, install it as dev dependencies (llike on sapper) and as dependecies but get this error :

 ```
 500
module is not defined

ReferenceError: module is not defined
    at /node_modules/lodash.tonumber/index.js?v=c404bec9:165:1
    at instantiateModule (/mnt/1C12BBAC12BB8972/Projects/Svelte/sveltekit/typescript/node_modules/vite/dist/node/chunks/dep-0776dd57.js:68919:166)
 ```

**Comment by bestguy** [comment]
Hi @kamalkech , thanks I haven't tested yet.  Will take a look at what the issue or tweaks needed are.

**Comment by bestguy** [comment]
Hi @kamalkech , can you please try to install `npm install sveltestrap@4.0.0-alpha.2` and confirm here if this fixes the issue?

I think it's an issue with es modules and updated to lodash-es.

**Comment by kamalkech** [comment]
@bestguy i found a solution by adding this line in vite config (svelte.config.cjs) :

```
optimizeDeps: { include: ['lodash.tonumber'] }
```

**Comment by bestguy** [comment]
Thanks for info @kamalkech 

I went ahead and removed lodash as a dependency, it was overkill for the need here.  Published as `sveltestrap@3.13.1`

**Comment by kamalkech** [comment]
greate ;)
